### PR TITLE
Added information about multitenant example

### PR DESCRIPTION
**Pull request by ardabeyazoglu**

### CODE REVIEW

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/create-sso-office-add-ins-nodejs.md](https://github.com/ardabeyazoglu/office-js-docs-pr/blob/patch-1/docs/develop/create-sso-office-add-ins-nodejs.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C8%5C22%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201808221632115855-337%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Wbn%2FNBpd8TvQiCRMQS031uobOqzbhYvfi7gD%2BmflJb0%3D&st=2018-08-22T16%3A28%3A23Z&se=2018-09-22T16%3A33%3A23Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/develop/create-sso-office-add-ins-nodejs.md [hunk 1]
```diff
@@ -29,9 +29,10 @@ This article walks you through the process of enabling single sign-on (SSO) in a
 1. Clone or download the repo at [Office Add-in NodeJS SSO](https://github.com/officedev/office-add-in-nodejs-sso). 
 
     > [!NOTE]
-    > There are two versions of the sample:  
+    > There are three versions of the sample:  
     > * The **Before** folder is a starter project. The UI and other aspects of the add-in that are not directly connected to SSO or authorization are already done. Later sections of this article walk you through the process of completing it. 
     > * The **Completed** version of the sample is just like the add-in that you would have if you completed the procedures of this article, except that the completed project has code comments that would be redundant with the text of this article. To use the completed version, just follow the instructions in this article, but replace "Before" with "Completed" and skip the sections **Code the client side** and **Code the server** side.
+    > * The **Completed Multitenant** version of the sample is same as **Completed**, except that the latter does not support multitenancy. If you are intended to support all microsoft accounts from different domains to use sso (onmicrosoft.com, outlook.com etc.), try this one.
```
Comment (AlexJerabek) [inline docs/develop/create-sso-office-add-ins-nodejs.md]: Just some minor things to match the Microsoft writing style guide:

Please change `The **Completed Multitenant** version of the sample is same as **Completed**, except that the latter does not support multitenancy.` to `The **Completed Multitenant** version is a completed sample that supports multitenancy.`

Comment (AlexJerabek) [inline docs/develop/create-sso-office-add-ins-nodejs.md]: Please change `If you are intended to support all microsoft accounts from different domains to use sso (onmicrosoft.com, outlook.com etc.), try this one.` to `Explore this sample if you intend to support Microsoft accounts from different domains with SSO.`

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/create-sso-office-add-ins-nodejs.md](https://github.com/ardabeyazoglu/office-js-docs-pr/blob/patch-1/docs/develop/create-sso-office-add-ins-nodejs.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C8%5C22%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201808221800242241-337%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=HbTEvAhjloF7ZdlIgR%2FzplX%2FBY9qNshLe%2Bm850Shzgg%3D&st=2018-08-22T17%3A56%3A35Z&se=2018-09-22T18%3A01%3A35Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by AlexJerabek** [comment]
Thank you again for this sample @ardabeyazoglu. Your changes to the docs will be merged into the live branch soon.

**Comment by ardabeyazoglu** [comment]
@AlexJerabek No problem. I've spent significant time to understand what is missing for multitenancy, at least others won't :)

**Pull request merged**